firmware_updater/bootloader/README.md

```diff
@@ -0,0 +1,10 @@
+# Selfbus Bootloader
+
+**The Bootloader  is used to update the firmware of Selfbus devices.**
+
+Information on usage [Wiki Selfbus Firmware-Updater](https://selfbus.org/wiki/software/tools/7-selfbus-bus-updater-tool).
+
+Flashing an Selfbus ARM-controller [Wiki Flashen der Module und Controller](https://selfbus.org/wiki/software/tools/8-how-to-flash-modules-and-controllers).
+
+### Filename convention
+- Files with **aprog** in filenames are for Devices with an non Selfbus "standard" programming button e.g. [Schaltaktor 6-fach strommessend 230V 16A 6TE](https://selfbus.org/wiki/devices/outputs/22-switching-actuator-6x-current-sensing-230v-16a-6mount-units).[README.md](..%2Fbootloaderupdater%2FREADME.md)
```

Comment:
What is the trailing README.md link about? Seems misplaced.

Reply:
Upsi. Thx.